docs/competition/teams/process-docs/sr2019-orientation-and-pit-rules.md

```diff
@@ -0,0 +1,81 @@
+## Welcome to Student Robotics 2019!
+
+## Pit Rules and Guidance
+
+The following rules must be observed at all times in the pits:
+
+1.	No battery charging. You must hand over your battery charger and spare batteries at reception. During the competition, visit Helpdesk (on the top floor, see map) to swap a discharged battery for a charged one.
+2.	No powered tools outside of the power tools area on the bottom floor (such as drills, soldering irons, etc.). **Power tools should only be used in the downstairs area.**
+3.	Keep your robot in your pit when possible – we may need to find you quickly.
+4.	Please avoid blocking or restricting aisles through pits.
+5.	Please refrain from running any wireless access points. If the provided network infrastructure isn't enough for teams, this issue should be raised to helpdesk.
+6. Only sealed food and drinks should be consumed in the pits area.
+
+## Network and Website Details
+
+A wireless network is available, connection details:
+
+* **SSID:** SR
+* **Password:** easyasabc
```

Comment:
usernames and passwords will be given to team leaders

Reply:
Disregarding as all orientation information is being stripped out